Packs/IdentityRecordedFuture/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+
+##### Recorded Future Identity
+
+- Added **reopen** arguments to the **recordedfuture-identity-playbook-alerts-update** command.
```

Comment:
```suggestion
- Added the **reopen** argument to the **recordedfuture-identity-playbook-alerts-update** command.
```

Reply:
1. yes, in the line 40 is how we get new argument and send it to backend.

2. updated the test to https://mockurl.com 

3. updated the release notes and reviewed all the dicts with access by key